Objects/Converters/ConverterNavisworks/ConverterNavisworks/ConverterNavisworks.cs

```diff
@@ -47,7 +47,7 @@ public ConverterNavisworks()
     public ReceiveMode ReceiveMode { get; set; }
 
 
-    public Document Doc { get; private set; }
+    public static Document Doc { get; private set; }
```

Comment:
Does this need to be static? It can usually be just accessed in the Converter via partial classes...